### PR TITLE
Enable GCS integration tests in Kokoro.

### LINKED ISSUE #640: Enable integration tests in Kokoro
The integration tests do not work in Kokoro because the credentials only work with access tokens, and we have a key file.  Disable the tests until we solve this, either improve the credential classes, or create the access token using an external too.

### PR DESCRIPTION
**Pull request by coryan**
Had to actually compile some of the integration tests with Bazel,
and cleanup the unnecessary debug output from the library too.
This fixes #640.

@mfschwartz since Matt is OOO, I do need your review on this one.

### CODE REVIEW

File: ci/kokoro/ubuntu/build.sh [hunk 1]
```diff
@@ -91,9 +91,6 @@ wget -q https://raw.githubusercontent.com/grpc/grpc/master/etc/roots.pem
 echo "    Getting cbt tool"
 export GOPATH="${KOKORO_ROOT}/golang"
 go get -u cloud.google.com/go/bigtable/cmd/cbt
-# TODO(#640) - disabled, the GCS library does not support credential files yet.
-# echo "    Getting python modules"
-# sudo python2 -m pip install httpbin
```
Comment (mfschwartz) [inline ci/kokoro/ubuntu/build.sh]: Why is this pip module no longer required? Seems unrelated to enabling integration tests.

Comment (coryan) [reply ci/kokoro/ubuntu/build.sh]: Note that it was all commented out before.  I thought we would need it for the integration tests, turns out we won't, so I removed the TODO and the commented out code.

**Comment by coryan** [comment]
PTAL.
